app/controllers/conference_registrations_controller.rb

```diff
@@ -33,8 +33,10 @@ def new
   end
 
   def show
-    @total_price = Ticket.total_price(@conference, current_user)
+    TicketPurchase.destroy_all(user_id: current_user.id, conference_id: @conference.id, paid: 'f')
```

Comment:
So you want to manipulate existing data that is there before your code goes live? We use rake tasks for this, not code in the app